lib/matplotlib/tests/test_pickle.py

```diff
@@ -89,6 +91,8 @@ def _generate_complete_test_figure(fig_ref):
     plt.subplot(3, 3, 9)
     plt.errorbar(x, x * -0.5, xerr=0.2, yerr=0.4)
 
+    fig_ref.align_ylabels()
```

Comment:
Add a comment about this testing the ability to pickle Grouper objects? Otherwise this seems like it is just thrown in there.

Reply:
sure, done.